File: java/simple_push.java
public void sendPostRequest(final String notifJson) {

    new AsyncTask(){

        @Override
        protected Void doInBackground(Object[] objects) {
            OkHttpClient client = new OkHttpClient();
            MediaType JSON = MediaType.parse("application/json; charset=utf-8");
            try {
                JSONObject jsonObject = new JSONObject(notifJson);


                RequestBody body = RequestBody.create(JSON, jsonObject.toString());
                Request request = new Request.Builder()
                    .url("https://panel.pushe.co/api/v1/notifications/")
                    .post(body)
                    .addHeader("content-type", "application/json; charset=utf-8")
                    .addHeader("Authorization", "Token YOUR_TOKEN") //use your token here as TOKEN
                    .addHeader("Accept", "application/json")
                    .build();

                client.newCall(request).enqueue(new Callback() {

                    @Override
                    public void onFailure(Call call, IOException e) {
                        Log.e("response", call.request().body().toString());

                    }

                    @Override
                    public void onResponse(Call call, Response response) throws IOException {
                        Log.e("response", response.body().string());
                    }

                });

            } catch (JSONException e) {
                Log.e("pushe", e.getMessage());
            }

            return null;
        }
    }.execute();

}

File: nodejs/simple_push.js
var request = require('request');

let TOKEN = "YOUR_TOKEN";

request.post(
    {
        uri: "https://api.pushe.co/v2/messaging/notifications/",
        method: "POST",
        headers: {
            "Content-Type": "application/json",
            "Accept": "application/json",
            "Authorization": "Token " + TOKEN,
        },
        body: JSON.stringify({
            "app_ids": ["YOUR_APPLICATION_ID"],
            "data": {
                "title": "This ia the title",
                "content": "This is the content",
            }
        }),
    },
    (error, response, body) => {
        console.log(`status_code: ${response.statusCode}`)
        console.log(`response: ${body}`);

        if (response.statusCode == 201){
            console.log("Success!");

            var data = JSON.parse(body);
            let report_url;

            if (data.hashed_id) {
                report_url = `https://pushe.co/report?id=${data.hashed_id}`;
            } else {
                report_url = "no report url for your plan";
            }

            console.log(`report_url: ${report_url}`);

            console.log(`notification id: ${data.wrapper_id}`);
        } else {
            console.log("failed!");
        }
    }
);
